<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget2</class>
 <widget class="QWidget" name="Widget2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnShowDialogErster">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>show dialogErster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHideMe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>hide me</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtLog">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
